fs/fs.go

```diff
@@ -433,6 +505,18 @@ func minor(devNumber uint64) uint {
 	return uint((devNumber & 0xff) | ((devNumber >> 12) & 0xfff00))
 }
 
+func (self *RealFsInfo) GetDeviceInfoByFsUUID(uuid string) (*DeviceInfo, error) {
+	deviceName := self.getDeviceNameFromFsUUID(uuid)
+	if deviceName == "" {
+		return nil, nil
```

Comment:
Can we return an error here instead of nil?  This eliminates the need check if *DeviceInfo is nil after calling in manager.GetFsInfoByFsUUID